clowdapp.yaml

```diff
@@ -317,6 +337,38 @@ objects:
           value: "secret_key_base"
         - name: SETTINGS__REPORT_DOWNLOAD_SSL_ONLY
           value: ${SETTINGS__REPORT_DOWNLOAD_SSL_ONLY}
+        - name: MAX_INIT_TIMEOUT_SECONDS
+          value: "${MAX_INIT_TIMEOUT_SECONDS}"
+
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: init-scripts
+  data:
+    abort_if_pending_migrations.sh: |
```

Comment:
Can we add these scripts to the image? That would avoid custom maps and mounts. Thanks.

Reply:
Using the ConfigMaps has the point of avoiding having to rebuild the image and adding those there :) I mean that's what I was trying to avoid.

Reply:
I know. The image is build nevertheless for each PR and merge.

Reply:
Working on moving the scripts to the image - once it's tested I'll ping you for re-review

Reply:
finally, it's green :)